server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	uidpb "github.com/buildbuddy-io/buildbuddy/proto/user_id"
```

Comment:
our style is to separate system imports, unaliased project imports and aliased project imports into separate groups

Reply:
Done